api/client/volume/client.go

```diff
@@ -441,7 +439,7 @@ func (v *volumeClient) Unquiesce(volumeID string) error {
 // CredsEnumerate enumerates configured credentials in the cluster
 func (v *volumeClient) CredsEnumerate() (map[string]interface{}, error) {
 	creds := make(map[string]interface{}, 0)
-	err := v.c.Get().Resource(credsPath).Do().Unmarshal(&creds)
+	err := v.c.Get().Resource(api.OsdCredsPath).Do().Unmarshal(&creds)
```

Comment:
Why is this change needed? These are the types of things that would be good to document to the reviewer on the PR.

Reply:
"OsdCredsPath" is already defined in api.go, no reason to define them locally here too.

Reply:
Cool